### DISCUSSION TITLE
Issue generating inpainted mesh

### DISCUSSION THREAD

**Discussion by jayarch** [thygate/stable-diffusion-webui-depthmap-script#455] [Q&A]
I can create a depth map but the 3D mesh and video aren't working. I deleted the models under 3D photo per a seemingly related question, but that didn't solve it. I also changed the file names per this thread : https://github.com/thygate/stable-diffusion-webui-depthmap-script/issues/371 and got the same result

Tried with GPU as well as CPU. 

Mac mini m2 pro / 16GB RAM

When I generate, I get a depth map but no mesh or video (takes only a minute) 

this is the output

DepthMap v0.4.7 (76a758c5)
Backbone: WEBUI
device: cpu
Loading model(s) ..
Loading model weights from  ./models/leres/res101.pth
initialize network with normal
loading the model from ./models/pix2pix/latest_net_G.pth
Computing output(s) ..
  0%|                                                                                                                                                              | 0/1 [00:00<?, ?it/s]wholeImage being processed in : 1344
Adjust factor is: 1.3233024691358024
Selecting patches ...
Target resolution:  (2746, 3557, 3)
Resulting depthmap resolution will be : (2160, 2798)
patches to process: 14
	 processing patch 0 / 13 | [   0    0 1762 1762]
	 processing patch 1 / 13 | [   0    0 1233 1233]
	 processing patch 2 / 13 | [   0  264 1233 1233]
	 processing patch 3 / 13 | [   0  529 1233 1233]
	 processing patch 4 / 13 | [   0  793 1233 1233]
	 processing patch 5 / 13 | [ 297   33 1167 1167]
	 processing patch 6 / 13 | [ 132 1189  969  969]
	 processing patch 7 / 13 | [  0 529 705 705]
	 processing patch 8 / 13 | [  0 793 705 705]
	 processing patch 9 / 13 | [   0 1057  705  705]
	 processing patch 10 / 13 | [   0 1321  705  705]
	 processing patch 11 / 13 | [264   0 705 705]
	 processing patch 12 / 13 | [529   0 705 705]
	 processing patch 13 / 13 | [1586 1057  705  705]
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:59<00:00, 59.68s/it]
Computing output(s) done.
Running 3D Photo Inpainting .. 
Loading edge model ..
local variable 'rgb_model' referenced before assignment, some issue with generating inpainted mesh
All done.

There is no mesh in the middle tab in the right column.

**Comment by Corwin-dev** [comment]
I had this same issue and managed to fix it myself. But notably, I also had an error during initialization of Automatic1111 saying "Inpaint import failed. Inpaint will not work.", so if you don't also have that error, this fix probably wont work.

Ultimately, moviepy needed to be downgraded to 1.0.3

`.venv\Scripts\activate`
`pip install moviepy==1.0.3`
